framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Modal/AbpModalFooterTagHelper.cs

```diff
@@ -1,8 +1,13 @@
-﻿namespace Volo.Abp.AspNetCore.Mvc.UI.Bootstrap.TagHelpers.Modal
+﻿using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Volo.Abp.AspNetCore.Mvc.UI.Bootstrap.TagHelpers.Modal
 {
+    [HtmlTargetElement("abp-modal")]
```

Comment:
` [HtmlTargetElement("abp-modal")]`
Is that intended? I don't see any reason for that, also that may cause bugs.

Reply:
I checked the references and `AbpModalFooterTagHelper` is only used in AbpModal.

Thought if buttons may be wanted to be used inside the modal body aswell.